### ISSUE TITLE
CLI iOS builds fail with BranchSDK.h:4:9: fatal error: 'Branch.h' file not found

### ISSUE THREAD

**Issue by bobrosoft** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution#633]
branch-cordova-sdk: 4.1.3
cordova: 9.0.0 (cordova-lib@9.0.1)
node: v10.16.0
Xcode: 11.2.1
@ionic/angular: 4.6.2
@ionic-native: 5.9.0

Hi there.
All worked ok when we was on branch-cordova-sdk 3.2.0 version before, but recently we've got "[Action Required] Please confirm your Play Store Referrer configuration" email and I wanted to update to latest SDK to fix that. I found issue #602 and was able to solve it by removing DerivedData with `rm -rf ~/Library/Developer/Xcode/DerivedData` but it works ONLY when I build project manually through Xcode which is **unacceptable way for us because we fully rely on automated CLI builds** and now it fails. All stuff was freshly installed.

CLI `cordova build ios` is failing now with next error: 
```
BranchSDK.h:4:9: fatal error:
      'Branch.h' file not found
#import "Branch.h"
        ^~~~~~~~~~
```

I'm wondering if that happens because you removed `use_frameworks` in latest versions? Why that was done?

A bit more complete log below:
```
CompileC /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/BranchSDK.o /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Plugins/branch-cordova-sdk/BranchSDK.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'Select Card App' from project 'Select Card App')
    cd /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios11.0-simulator -fmessage-length=116 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=c99 -fobjc-arc -fmodules -gmodules -fmodules-cache-path=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Wunreachable-code -Wno-implicit-atomic-properties -Wno-objc-interface-ivars -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version=2 -fobjc-legacy-dispatch -index-store-path /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Index/DataStore -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-generated-files.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-own-target-headers.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-all-non-framework-target-headers.hmap -ivfsoverlay /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/all-product-headers.yaml -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-project-headers.hmap -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/usr/local/lib/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/iphonesimulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch/Branch.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift/CryptoSwift.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources-normal/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Pods/EMLSDK -include /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/sharedpch/SharedPrecompiledHeaders/234946451765851508/Select\ Card\ App-Prefix.pch -MMD -MT dependencies -MF /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/BranchSDK.d --serialize-diagnostics /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/BranchSDK.dia -c /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Plugins/branch-cordova-sdk/BranchSDK.m -o /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/BranchSDK.o
In file included from /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select Card App/Plugins/branch-cordova-sdk/BranchSDK.m:1:
/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select Card App/Plugins/branch-cordova-sdk/BranchSDK.h:4:9: fatal error:
      'Branch.h' file not found
#import "Branch.h"
        ^~~~~~~~~~
1 error generated.

CompileC /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate.o /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Classes/AppDelegate.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'Select Card App' from project 'Select Card App')
    cd /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios11.0-simulator -fmessage-length=116 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=c99 -fobjc-arc -fmodules -gmodules -fmodules-cache-path=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Wunreachable-code -Wno-implicit-atomic-properties -Wno-objc-interface-ivars -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version=2 -fobjc-legacy-dispatch -index-store-path /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Index/DataStore -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-generated-files.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-own-target-headers.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-all-non-framework-target-headers.hmap -ivfsoverlay /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/all-product-headers.yaml -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-project-headers.hmap -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/usr/local/lib/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/iphonesimulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch/Branch.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift/CryptoSwift.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources-normal/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Pods/EMLSDK -include /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/sharedpch/SharedPrecompiledHeaders/234946451765851508/Select\ Card\ App-Prefix.pch -MMD -MT dependencies -MF /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate.d --serialize-diagnostics /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate.dia -c /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Classes/AppDelegate.m -o /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate.o

CompileC /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate+BranchSdk.o /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Plugins/branch-cordova-sdk/AppDelegate+BranchSdk.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'Select Card App' from project 'Select Card App')
    cd /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios11.0-simulator -fmessage-length=116 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=c99 -fobjc-arc -fmodules -gmodules -fmodules-cache-path=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/Users/bobr/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Wunreachable-code -Wno-implicit-atomic-properties -Wno-objc-interface-ivars -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version=2 -fobjc-legacy-dispatch -index-store-path /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Index/DataStore -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-generated-files.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-own-target-headers.hmap -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-all-non-framework-target-headers.hmap -ivfsoverlay /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/all-product-headers.yaml -iquote /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Select\ Card\ App-project-headers.hmap -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator/usr/local/lib/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/include -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/UninstalledProducts/iphonesimulator/include -I/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch/Branch.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift/CryptoSwift.framework/Headers -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources-normal/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources/x86_64 -I/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/DerivedSources -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/emulator -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/Branch -F/Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Products/Debug-iphonesimulator/CryptoSwift -F/Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Pods/EMLSDK -include /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/build/sharedpch/SharedPrecompiledHeaders/234946451765851508/Select\ Card\ App-Prefix.pch -MMD -MT dependencies -MF /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate+BranchSdk.d --serialize-diagnostics /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate+BranchSdk.dia -c /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Plugins/branch-cordova-sdk/AppDelegate+BranchSdk.m -o /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/AppDelegate+BranchSdk.o

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/bobr/Library/Developer/Xcode/DerivedData/Select_Card_App-bubqjfsymvhakkfufnkehyyazdqt/Build/Intermediates.noindex/Select\ Card\ App.build/Debug-iphonesimulator/Select\ Card\ App.build/Objects-normal/x86_64/BranchSDK.o /Users/bobr/Documents/My/Work/cleverbug/repos/clevergift-love2shop/platforms/ios/Select\ Card\ App/Plugins/branch-cordova-sdk/BranchSDK.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
xcodebuild: Command failed with exit code 65
```

**Comment by bobrosoft** [comment]
Forked the repo and **temporarily** _fixed_ it with switching back to `use_frameworks` with next commit https://github.com/bobrosoft/cordova-ionic-phonegap-branch-deep-linking-attribution/commit/72f11eb9f88b0412cc27c59a0ab9247465b4b05b. Seems working. 
```
"branch-cordova-sdk": "git+ssh://git@github.com/bobrosoft/cordova-ionic-phonegap-branch-deep-linking-attribution.git",
```
Don't forget to 
```
rm -rf ~/Library/Developer/Xcode/DerivedData
```

**Comment by codinronan** [comment]
@bobrosoft what versions of cordova-ios and CocoaPods are you using? The need to use ‘use_frameworks’ indicates one of those two is out of date. I’ve got CLI builds working with this version and IIRC @echo-branch  indicated they were working for him too after updating.

Saying that we’re working on backporting the relevant changes to the 3.x releases as well if you don’t want to use CocoaPods.

**Comment by bobrosoft** [comment]
@codinronan 
cordova-ios: 5.0.1
pod: 1.7.5 (there were issues with CDN 6 months ago when I tried to update it)

So, that means I just need to update `pod` to latest?

**Comment by codinronan** [comment]
@bobrosoft correct, that error is *specifically* fixed by using CocoaPods >= 1.8.4 - the reason is that it no longer locally resolves the file, it uses the path relative to the CDN download, which means that each CocoaPod doesn't need its own configuration path (which presented issues in the past).

**Comment by bobrosoft** [comment]
@codinronan good to know, thanks for explanation! Closing...
